app/src/main/java/app/revanced/integrations/youtube/patches/misc/requests/StreamingDataRequest.java

```diff
@@ -160,6 +165,11 @@ private static boolean isLiveStream(@NonNull JSONObject playerResponse) {
         return false;
     }
 
+    private static boolean isLiveStream(String streamingData) {
+        // Check the source parameter in streamingData
+        return streamingData.contains("yt_live_broadcast") || streamingData.contains("yt_premiere_broadcast");
```

Comment:
Use this instead:

Stream.of(
    "yt_live_broadcast",
    "yt_premiere_broadcast"
)
.anyMatch(streamingData::contains)